jax_rocm_plugin/build/rocm/tools/get_rocm.py

```diff
@@ -342,14 +383,16 @@ def parse_args():
 
 
 def main():
+    """Installs ROCm at /opt/rocm on your system"""
     args = parse_args()
     if args.rocm_version == "latest":
         try:
             rocm_version = latest_rocm()
             print("Latest ROCm release: %s" % rocm_version)
-        except Exception:
+        except URLError:
```

Comment:
I think this still needs to be Exception, because URLError is only one way it can fail. The block is meant to catch anything that goes wrong in the block and return an error message rather than stack trace.